west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: a0eba5b06796254ae6ccd622566c3e7b831a3559
+      revision: pull/1028/head
```

Comment:
Please update SHA: `62f7968a7a2d8932048f9c0b4c46b2ada15ed3cd` (and rebase).

Reply:
Updated.